WordPress/Jetpack/Classes/ViewRelated/WordPress Migration/Views/MigrationHeaderView.swift

```diff
@@ -0,0 +1,90 @@
+import UIKit
+
+final class MigrationHeaderView: UIView {
+
+    // MARK: - Views
+
+    let imageView = UIImageView()
+
+    let titleLabel: UILabel = {
+        let label = UILabel()
+        label.font = Constants.titleFont
+        label.numberOfLines = 0
+        label.adjustsFontForContentSizeCategory = true
+        return label
+    }()
+
+    let primaryDescriptionLabel: UILabel = {
+        let label = UILabel()
+        label.font = Constants.primaryDescriptionFont
+        label.numberOfLines = 0
+        label.adjustsFontForContentSizeCategory = true
+        return label
+    }()
+
+    let secondaryDescriptionLabel: UILabel = {
+        let label = UILabel()
+        label.font = Constants.secondaryDescriptionFont
+        label.numberOfLines = 0
+        label.adjustsFontForContentSizeCategory = true
+        return label
+    }()
+
+    // MARK: - Init
+
+    init() {
+        super.init(frame: .zero)
+        self.setup()
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.setup()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setup() {
+        // Set subviews
+        let labelsStackView = Self.verticalStackView(arrangedSubviews: [titleLabel, primaryDescriptionLabel, secondaryDescriptionLabel])
+        labelsStackView.spacing = Constants.labelsSapcing
```

Comment:
Nit: Typo

```suggestion
        labelsStackView.spacing = Constants.labelsSpacing
```

Reply:
Resolved in Resolved in https://github.com/wordpress-mobile/WordPress-iOS/pull/19533/commits/1d2a3a9a756e51e0fb0c0a8f1b07519c6af4d14d